kubejs/server_scripts/constants/metal_properties.js

```diff
@@ -0,0 +1,194 @@
+//priority: 1010
+
+// Used for correcting tags for Blue Skies materials
+const blueSkiesMetals = ['falsite', 'ventium', 'horizonite'];
+
+// Properties for Metal processing
+const metalProperties = {
```

Comment:
Would this not be easier to work with as an array, where the metal name is a property in each entry?